### PR TITLE
[Breaking change] policygen: support multiple state paths

### PR DESCRIPTION
**Pull request by xingao267**
Fixes https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/issues/470

### CODE REVIEW

File: internal/policygen/policygen.go [hunk 1]
```diff
@@ -38,7 +40,7 @@ const (
 // RunArgs is the struct representing the arguments passed to Run().
 type RunArgs struct {
 	ConfigPath string
-	StatePath  string
+	StatePaths string
```
Comment (umairidris) [inline internal/policygen/policygen.go:43]: why not make this a list of string and do the split in the caller?

**Comment by umairidris** [comment]
When are we going to do the next major release? Maybe we should keep state_path around till then and just append it to the state paths list?

File: internal/policygen/policygen.go (hunk 1 above)
Comment (xingao267) [reply internal/policygen/policygen.go:43]: I'm ok either way. I feel using a comma separated list makes the command line a bit more compact and is what `--templates` is doing tin the tfengine command line.

**Comment by xingao267** [comment]
> When are we going to do the next major release? Maybe we should keep state_path around till then and just append it to the state paths list? Then we can remove it when there is more a necessity for a breaking change.

We are releasing first week of each month, so technically this week or next week. I can hold this PR and merge until we are making a release later this week.

File: internal/policygen/policygen.go (hunk 1 above)
Comment (umairidris) [reply internal/policygen/policygen.go:43]: tfengine seems to do it in the main as well: https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/cmd/tfengine/main.go#L67

I would vote for doing this in main. Seems odd to have a  StatePaths (plural) that's a string (singular). list of strings is more explicit.

Comment (xingao267) [reply internal/policygen/policygen.go]: Done. Thought you were referring to custom flag. Moved to main.go

**Pull request merged**